paper-radio-button.html

```diff
@@ -42,16 +42,17 @@
 @element paper-radio-button
 -->
 
-<dom-module id="paper-radio-button">
-  <style is="custom-style">
-    * {
-      --paper-radio-button-unchecked-color: var(--primary-text-color);
-      --paper-radio-button-unchecked-ink-color: var(--primary-text-color);
+<style is="custom-style">
+  * {
+    --paper-radio-button-unchecked-color: var(--primary-text-color);
+    --paper-radio-button-unchecked-ink-color: var(--primary-text-color);
 
-      --paper-radio-button-checked-color: var(--default-primary-color);
-      --paper-radio-button-checked-ink-color: var(--default-primary-color);
-    }
-  </style>
+    --paper-radio-button-checked-color: var(--default-primary-color);
+    --paper-radio-button-checked-ink-color: var(--default-primary-color);
+  }
```

Comment:
Are we not putting it inline with the variable declaration in the element style?